vm/ieleApi.ml

```diff
@@ -27,15 +27,14 @@ let get_block blocknumber =
 
 let personal_newAccount () =
   let acct = Cryptokit.Random.string random 20 in
-  let key = to_hex (Bytes.of_string acct) in
+ `String (to_hex (Bytes.of_string acct))
```

Comment:
Why does `personal_newAccount` not add the account to the state now? Is this meant for simple wallets?

Reply:
It's taken care of by setChainParams. The account isn't added to the state because it has the same state as the default, non-existent account.

Reply:
Ok, having the same state as a default non-existing record seems sufficient to justify this. What does setChainParams have to do with this?